src/main/java/com/conveyal/r5/analyst/cluster/AnalysisWorkerTask.java

```diff
@@ -111,9 +111,9 @@ public abstract class AnalysisWorkerTask extends ProfileRequest {
 
     /**
      * If this is set to a value above zero, report the amount of time needed to reach the given number of
```

Comment:
Javadoc here (and above for `includeTemporalDensity`) should be updated (as in `AnalysisRequest`)

Reply:
Could still use an update:
```
     * If set, and if includeTemporalDensity is true, results will 
     * include the amount of time needed to reach each threshold 
     * number of opportunities (known technically as "dual access").
```

Reply:
Updated comment, thanks 👍